src/ckb-daemon/usb_mac.c

```diff
@@ -179,14 +179,14 @@ int _nk95cmd(usbdevice* kb, uchar bRequest, ushort wValue, const char* file, int
 void os_sendindicators(usbdevice* kb){
     void *ileds;
     ushort leds;
-    if(kb->fwversion >= 0x300) {
+    if(kb->fwversion >= 0x300 || IS_V3_OVERRIDE(kb)) {
         leds = (kb->ileds << 8) | 0x0001;
         ileds = &leds;
     }
     else {
         ileds = &kb->ileds;
     }
-    IOUSBDevRequestTO rq = { 0x21, 0x09, 0x0200, 0x00, (kb->fwversion >= 0x300 ? 2 : 1), ileds, 0, 500, 500 };
+    IOUSBDevRequestTO rq = { 0x21, 0x09, 0x0200, 0x00, ((kb->fwversion >= 0x300 || IS_V3_OVERRIDE(kb)) ? 2 : 1), ileds, 0, 500, 500 };
```

Comment:
Likewise, this should be refactored.

Reply:
I don't see a point in refactoring code that we'll most likely get rid of soon. We don't get direct control over devices on macOS anyway, so I doubt this even gets executed.